### ISSUE TITLE
Add Animal Telemetry Network Tile to ioos.us

### ISSUE THREAD

**Issue by BeckyBaltes** [ioos/ioos-us#144]
We are ready to add a new tile for ATN. I can discuss deadlines for this task. We don't have a hard deadline on our end but we should set a target for at least this Fall.  The content is [here ](https://docs.google.com/document/d/1rumqcAoitYDKFtE6HvDAw4_2MgS2l52jTwAQpZf-BC4/edit)on a google doc. I can share it with whomever needs it, and a call may be easiest to discuss layout, but I'm envisioning a similar layout to the glider DAC and HFR tiles.

**Comment by kellydesent** [comment]
Hi Becky, I will be working on this task. Can you please share the google doc with my email, Kelly.Desent@rpsgroup.com. Thanks.

**Comment by BeckyBaltes** [comment]
Done. Thanks

**Comment by BeckyBaltes** [comment]
@kellydesent Any idea when this will be done?

**Comment by kellydesent** [comment]
@BeckyBaltes I have all of the content up and the new tile added. I am waiting for Bob to deploy the new atn site on a dev link. I will get back to you soon on an estimate for that. Once I get the link, I will send it over and we can go over changes you would like made. Once it is final, Bob will deploy it to the atn.ioos.us site. I will be in touch soon, thanks.

**Comment by kellydesent** [comment]
@BeckyBaltes Here is the dev link: https://atn-dev.ioos.us/
Please let me know what changes you would like made, including any styling and layout updates.

**Comment by BeckyBaltes** [comment]
Thanks @kellydesent. It looks good to me, but I've sent it to the SME to review. As soon as Bill gets back to me, you can launch it.

**Comment by kellydesent** [comment]
Ok sounds good. Under "About", the link for "Documents" and "Contact Us" directs back to the homepage, no matter what site your on, including this one: https://hfradar.ioos.us/. Do you have something else that should be there? I wasn't sure if it might have been missed in testing or if we are still waiting on content for that.

**Comment by BeckyBaltes** [comment]
@kellydesent Talked with Bill and here are the minor edits for the ATN page.

1. At the top of the page title change from **IOOS** Animal Tel. Network to **US** Animal Tel. Network.
2. When you hover over the map, change the text from "Click here to see the map" to "Click here to enter the portal"
3. Delete the "Data Vis and Access" and sub header content because it's not worthwhile enough to keep for now.
4. Change "Data Provider Tools" to "Data Portal FAQs"

For the About Us section:
Can you use the same info from the main ioos.us page?  If so, I'd like to just update that to include ATN and HFR.  I can start a separate issue with the updates for the About tab.

**Comment by BeckyBaltes** [comment]
@kellydesent One addition.

Please add this document to the documentation list:
Name: Ocean Observations Using Tagged Animals
[ATN Article.pdf](https://github.com/ioos/ioos-us/files/1468184/ATN.Article.pdf)

**Comment by kellydesent** [comment]
Hi, I have made these changes. We just need to deploy them onto the atn.ioos.us site. As for the about us content, if you could create a new issue, that would be great. I will get in contact with Bob to see what we need to do for that. I will let you know. Thanks.

**Comment by Bobfrat** [comment]
@BeckyBaltes Are we ok to deploy new ioos.us with the ATN tile pointing to atn.ioos.us?

**Comment by BeckyBaltes** [comment]
Yes please.

**Comment by Bobfrat** [comment]
Ok this has been deployed. you should see the ATN tile now

**Comment by BeckyBaltes** [comment]
I see the ATN tile, but I think there was suppose to be a picture associated with it.  Can you add that to the background of the tile? The picture is in the google doc I shared with @kellydesent.

**Comment by kknee** [comment]
@kellydesent can you figure out what happened to the photo please?

**Comment by Bobfrat** [comment]
All set Fixed here, will be deployed in a couple mins
https://github.com/ioos/ioos-us/pull/160

**Comment by BeckyBaltes** [comment]
That looks good. In reviewing the content Bill provided, we were suppose to include the logos of the participating agencies/groups. Can you add those under the portal entry and above the Resources?

**Comment by kellydesent** [comment]
@BeckyBaltes I will add the logos in right now.

**Comment by kellydesent** [comment]
@BeckyBaltes This is completed. Just waiting for it to be deployed. Thank you.

**Comment by BeckyBaltes** [comment]
Looks good to me. I think we can close this if you're ready.
